config/nvblox/specializations/nvblox_isaac_sim.yaml

```diff
@@ -1,3 +0,0 @@
-/**:
-  ros__parameters:
```

Comment:
why are these lines deleted? did you mean to delete the whole file instead?

Reply:
I did